subsys/dult/Kconfig

```diff
@@ -148,6 +148,58 @@ config DULT_USER
 	help
 	  Add DULT user source files.
 
+config DULT_MOTION_DETECTOR
+	bool "Enable DULT Motion detector feature"
+	help
+	  The DULT Motion detector requests the device to ring if the accessory separated
+	  from owner is moving. For more details see
+	  https://www.ietf.org/archive/id/draft-detecting-unwanted-location-trackers-01.html#name-motion-detector.
+
+if DULT_MOTION_DETECTOR
+
+config DULT_MOTION_DETECTOR_TEST_MODE
+	bool "Enable DULT Motion detector test mode"
+	help
+	  The DULT Motion detector test mode allows to configure motion detector parameters
+	  for testing purposes. Those values are defined in the DULT specification and should
+	  not be changed in the production code.
+
+config DULT_MOTION_DETECTOR_SEPARATED_UT_SAMPLING_RATE1
+	int "Sampling rate in milliseconds when motion detector is enabled in separated state" if DULT_MOTION_DETECTOR_TEST_MODE
+	default 10000
+
+config DULT_MOTION_DETECTOR_SEPARATED_UT_SAMPLING_RATE2
+	int "Motion detector sampling rate in milliseconds when movement is detected in separated state" if DULT_MOTION_DETECTOR_TEST_MODE
+	default 500
+
+config DULT_MOTION_DETECTOR_SEPARATED_UT_BACKOFF_PERIOD
+	int "Period in minutes to disable motion detector if accessory is in separated state" if DULT_MOTION_DETECTOR_TEST_MODE
+	default 360
+
+config DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MIN
+	int "Minimum time span in minutes in separated state before enabling motion detector" if DULT_MOTION_DETECTOR_TEST_MODE
+	default 480
+
+config DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MAX
+	int "Maximum time span in minutes in separated state before enabling motion detector" if DULT_MOTION_DETECTOR_TEST_MODE
+	default 1440
+
```

Comment:
Can you override the defaults for these parameters when the DULT_MOTION_DETECTOR_TEST_MODE is enabled so these values are test-friendly and you don't have to wait hours to test the motion detector algorithm?

Also, you can make the UT timeout min to be equal to the max, so that there is no randomness in the test mode.

The idea here is that the user enables in their project just one option:

`CONFIG_DULT_MOTION_DETECTOR_TEST_MODE=y`

and they can easily test how the motion detection feature works.

Here is my suggestion regarding the values:

```suggestion
config DULT_MOTION_DETECTOR_SEPARATED_UT_BACKOFF_PERIOD
	int "Period in minutes to disable motion detector if accessory is in separated state" if DULT_MOTION_DETECTOR_TEST_MODE
	default 2 if DULT_MOTION_DETECTOR_TEST_MODE
	default 360

config DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MIN
	int "Minimum time span in minutes in separated state before enabling motion detector" if DULT_MOTION_DETECTOR_TEST_MODE
	default 3 if DULT_MOTION_DETECTOR_TEST_MODE
	default 480

config DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MAX
	int "Maximum time span in minutes in separated state before enabling motion detector" if DULT_MOTION_DETECTOR_TEST_MODE
	default DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MIN if DULT_MOTION_DETECTOR_TEST_MODE
	default 1440

```